src/sciline/visualize.py

```diff
@@ -104,13 +105,24 @@ def get_subgraph_name(name: str) -> str:
 
 def _add_subgraph(graph: FormattedGraph, dot: Digraph, subgraph: Digraph) -> None:
     for p, (p_name, args, ret) in graph.items():
-        subgraph.node(
-            ret.name, ret.name, shape='box3d' if ret.collapsed else 'rectangle'
-        )
+        unsatisfied = f'{_qualname(HandleAsComputeTimeException.handle_unsatisfied_requirement)}.<locals>.unsatisfied_sentinel'  # noqa: E501
+        if p_name == unsatisfied:
+            subgraph.node(
+                ret.name,
+                ret.name,
+                shape='box3d' if ret.collapsed else 'rectangle',
+                color='red',
+                fontcolor='red',  # Set text color to red
```

Comment:
Can you add another visual hint in addition to colour? E.g. make the border dashed.

Reply:
Done